walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/handlers/OpenIdRequestHandlerTest.kt

```diff
@@ -338,7 +337,7 @@ class OpenIdRequestHandlerTest {
         assertThat((actualOpenIdRequest as ManifestIssuanceRequest).verifiedIdStyle).isInstanceOf(
             BasicVerifiedIdStyle::class.java
         )
-        assertThat((actualOpenIdRequest.verifiedIdStyle as BasicVerifiedIdStyle).logo).isNull()
+        assertThat((actualOpenIdRequest.verifiedIdStyle as BasicVerifiedIdStyle).verifiedIdLogo).isNull()
```

Comment:
thinking about this a little more in these tests. Can the property name on VerifiedIdStyle still be logo but the class name be VerifiedIdLogo?